app/components/avo/views/resource_index_component.rb

```diff
@@ -32,7 +32,7 @@ def initialize(
 
   def title
     if @reflection.present?
-      return field.plural_name if field.present?
+      return name if field.present?
```

Comment:
Can't we return `field.name` here? It seems that it fixes everything.

```ruby
      def name
      	# the default is pluralized by the declaration
        default = @id.to_s.humanize(keep_id_suffix: true)

				# returns the name if it's set
        return @name if @name.present?

				# get's it from the translation if that's set
        return t(translation_key, count: 1, default: default).capitalize if translation_key

        default
      end
```


I think that fixes it without requiring changes to the other files.


```suggestion
      return field.name if field.present?
```

Reply:
That was my [first try](https://github.com/avo-hq/avo/pull/926/commits/e9f17a4611d83086d412462cddbc6f14461cf058) but unfortunately that have an issue. Imagine we have a field without a custom name but it has a translation_key configured. In this case we want to return ... `translation_key, count:2` .... by default. That's why i make that verify, if name is present return singular name, else return plural default

Reply:
Got it! Makes sense!

Thank you!